.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,12 @@
+name: testing-PR
+on:
+  pull_request:
+    branches: [main]
```

Comment:
This will only work on the `main` branch. This PR should allow GitHub Actions to run for every PR to be able to test if the code is working.